doc/source/api/visualization/graphics_window.rst

```diff
@@ -0,0 +1,32 @@
+.. _ref_graphics_window:
+
+GraphicsWindow
+==============
+ 
+.. autopostdoc:: ansys.fluent.visualization.graphics.graphics_window.GraphicsWindow
+
+Creates a window to displays graphics. You can add graphics objects like
+mesh, surface or plots to this window and then display it.
```

Comment:
```suggestion
mesh, surface, or plot to this window and then display it.
```

Reply:
(I think we eliminated the Oxford comma requirement for PyFluent docs.)

The statement puts plots in the plural while the other nouns are singular -- is that intentional?